src/sys/ioctl/platform/linux.rs

```diff
@@ -1,12 +1,27 @@
-pub const NRBITS: u32 = 8;
-pub const TYPEBITS: u32 = 8;
+/// The datatype used for the ioctl number
+#[cfg(any(target_os = "android", target_env = "musl"))]
+#[doc(hidden)]
+pub type ioctl_num_type = ::libc::c_int;
```

Comment:
Really?  Musl defines the request as an int, even when it's running on Linux?  Baffling!